fenix/app/src/main/java/org/mozilla/fenix/shopping/ReviewQualityCheckBottomSheetStateFeature.kt

```diff
@@ -12,24 +12,24 @@ import org.mozilla.fenix.shopping.store.ReviewQualityCheckState
 import org.mozilla.fenix.shopping.store.ReviewQualityCheckStore
 
 /**
- * View-bound feature that requests the bottom sheet to be expanded when the store state changes
- * from [ReviewQualityCheckState.Initial] to [ReviewQualityCheckState.NotOptedIn].
+ * View-bound feature that requests the bottom sheet state to be changed to expanded or collapsed when
+ * the store state changes from [ReviewQualityCheckState.Initial] to [ReviewQualityCheckState.NotOptedIn].
  *
  * @param store The store to observe.
- * @property onRequestStateExpanded Callback to request the bottom sheet to be expanded.
+ * @property onRequestStateUpdate Callback to request the bottom sheet to be updated.
  */
 class ReviewQualityCheckBottomSheetStateFeature(
     store: ReviewQualityCheckStore,
-    private val onRequestStateExpanded: () -> Unit,
+    private val onRequestStateUpdate: (expanded: Boolean) -> Unit,
 ) : AbstractBinding<ReviewQualityCheckState>(store) {
     override suspend fun onState(flow: Flow<ReviewQualityCheckState>) {
         val initial = Pair<ReviewQualityCheckState?, ReviewQualityCheckState?>(null, null)
         flow.scan(initial) { acc, value ->
             Pair(acc.second, value)
         }.filter {
-            it.first is ReviewQualityCheckState.Initial && it.second is ReviewQualityCheckState.NotOptedIn
+            it.first is ReviewQualityCheckState.Initial
         }.collect {
-            onRequestStateExpanded()
+            onRequestStateUpdate(it.second is ReviewQualityCheckState.NotOptedIn)
```

Comment:
Here when it's `OptedIn` the sheet is expanded but this logic is only for expanding to onboarding state.